python/foglamp/services/core/routes.py

```diff
@@ -64,6 +64,7 @@ def setup(app):
     app.router.add_route('GET', '/foglamp/category/{category_name}/children', api_configuration.get_child_category)
     app.router.add_route('DELETE', '/foglamp/category/{category_name}/children/{child_category}', api_configuration.delete_child_category)
     app.router.add_route('DELETE', '/foglamp/category/{category_name}/parent', api_configuration.delete_parent_category)
+    app.router.add_route('PUT', '/foglamp/category/{category_name}/value', api_configuration.update_configuration_item_bulk)
```

Comment:
I think, We should drop `/value` from this url.

Reply:
Agreed

Reply:
Done